test/integration_tests/test_print_files.py

```diff
@@ -302,6 +302,58 @@ def test_P_OR_H4(sftp_client):
             '2|english_qid||||Mr|Test|McTest|123 Fake Street|Duffryn||Newport|NPXXXX|P_OR_H4\n'))
 
 
+def test_P_LP_HL1(sftp_client):
+    # Given
+    icl1e_messages, _ = build_test_messages(ICL_message_template, 1, 'P_LP_HLX', 'P_LP_HL1', uac=False)
```

Comment:
This is using the template for initial contact letters, I think it would be worth creating a new one with the extra data we expect for these pack codes to make sure things like title/forename/surname get included in the file

Reply:
Done